bench/wtperf/smoke.sh

```diff
@@ -1,4 +1,53 @@
 #! /bin/sh
 
 # Smoke-test wtperf as part of running "make check".
-./wtperf -O `dirname $0`/runners/small-lsm.wtperf -o "run_time=20"
+#./wtperf -O `dirname $0`/runners/small-lsm.wtperf -o "run_time=20"
+
+# Run something with a lot of random options so we test for memory leaks
+# with more than simple cases.
+args=""
+args="$args async_threads=4,"
+args="$args checkpoint_interval=10,"
+args="$args checkpoint_threads=1,"
+args="$args checkpoint_stress_rate=100,"
+args="$args compact=true,"
+args="$args compression=snappy,"
```

Comment:
The previous smoke test didn't use any compression. With snappy enabled here the test is erroring when --enable-snappy isn't set within the pull req tester